Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM python:3.7.2-alpine
+FROM python:3.9-slim
 
-RUN pip install --upgrade pip
-RUN apk add gcc musl-dev build-base postgresql-libs postgresql-dev
+RUN useradd -ms /bin/bash kvuser
+USER kvuser
+WORKDIR /home/kvuser
 
-RUN adduser -D worker
-USER worker
-WORKDIR /home/worker
+ENV PATH="/home/kvuser/.local/bin:${PATH}"
 
-ENV PATH="/home/worker/.local/bin:${PATH}"
-COPY --chown=worker:worker requirements.txt requirements.txt
-RUN pip install --user -r requirements.txt
+COPY --chown=kvuser:kvuser requirements.txt requirements.txt
+RUN python3 -m pip install --upgrade pip
```

Comment:
This uses only extra space in the image and in the future may lead to breakage
```suggestion
```

Reply:
removed string RUN python3 -m pip install --upgrade pip
recreated and uploaded to the ECR new image